cookietemple/lint/template_linter.py

```diff
@@ -649,6 +661,43 @@ def check_missing_sections(self, parsed_sections) -> bool:
             self.linter_ctx.passed.append(('general-7', 'All required cookietemple.cfg sections were found!'))
             return True
 
+    def lint_sync_blacklisted_section(self, section_items) -> bool:
+        """
+        Lint the 'sync_files_blacklisted' section according to the following rules:
+
+        1.) 'sync_files_blacklisted' should contain at least the 'CHANGELOG.rst' file (excluding it from syncing to avoid PR updates)
+
+        2.) If the project is a Python project, there should be: - a 'requirements = requirements.txt'
+                                                                 - and a 'requirements_dev = requirements_dev.txt' key-value pair
+
+        3.) If the project is a Java project: Either a 'gradle_build = gradle.build' (currently only Cli Java) or 'pom = pom.xml' must be present (GUI Java)
+
+        :return: Whether linting passed or not
+        """
+        # linting a Python project
```

Comment:
Honestly, for consistency reasons I feel like these kind of things should be in the respective implementing linter.

We will refactor this anyways, but for now I would try to aim for the following:

Just like check_files_exist the TemplateLinter should just provide the function with the signature `def verify_blacklisted_sync_files(list_files: list[str])`.

The sublinter should then implement the function

Reply:
I agree, but like u said, we will refactor this in the next time, so I'd rather go with this for now, since most code will be refactored.

So I'd put my time into other issues and fix this with our "big" refactoring.

Opinion @Zethson ?

Reply:
I would absolutely aim for consistency for now.
It will take a month or so at least until we can refactor this whole mess.